<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="showPasswdChecked" stdset="0">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="1">
        <widget class="QLineEdit" name="prevBlockText"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="pubkLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="resultTool">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="balanceText"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="prvkBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="genBtn">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QWidget" name="resultWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_3"/>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="addrLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="seedEdit">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="prvkText"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="toolsTabs"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>seedEdit</tabstop>
  <tabstop>prvkText</tabstop>
  <tabstop>prvkBtn</tabstop>
  <tabstop>balanceText</tabstop>
  <tabstop>prevBlockText</tabstop>
  <tabstop>genBtn</tabstop>
  <tabstop>toolsTabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
